src/main/java/edu/wpi/cs3733/D22/teamC/DBManager.java

```diff
@@ -146,4 +148,35 @@ public void initializeLocationTable() {
                 emptyTables
         );
     }
+    /**
+    Initialize Service Request Table
+     */
+    public void initializeSRTable() {
+        initializeTable("SERVICE_REQUESTS", "CREATE TABLE SERVICE_REQUESTS (\n" +
+                                                                        "    REQUESTID char(10),\n" +
+                                                                        "    CREATORID char(10),\n" +
+                                                                        "    ASSIGNEEID char(10),\n" +
+                                                                        "    LOCATIONID char(10),\n" +
+                                                                        "    CREATIONTIMESTAMP DATETIME,\n" +
+                                                                        "    STATUS  varchar2(50),\n" +
+                                                                        "    PRIORITY   varchar2(50),\n" +
+                                                                        "    REQUESTTYPE varchar2(50),\n" +
+                                                                        "    DESCRIPTION varchar2(150),\n" +
+                                                                        "    CONSTRAINT PK_REQUESTID PRIMARY KEY (REQUESTID))\n"
```

Comment:
Are any of these **\n** newline characters necessary for the setup of the table? initializeLocationTable doesn't use them.